src/transformers/models/owlvit/processing_owlvit.py

```diff
@@ -128,13 +134,23 @@ def __call__(self, text=None, images=None, padding="max_length", return_tensors=
             encoding["input_ids"] = input_ids
             encoding["attention_mask"] = attention_mask
 
+        if query_images is not None:
+            encoding = BatchEncoding()
+            query_pixel_values = self.feature_extractor(
+                query_images, return_tensors=return_tensors, **kwargs
+            ).pixel_values
```

Comment:
Nit, fits in one line.

Reply:
It doesn't fit in one line, running `make fixup` corrects it